packages/kbn-journeys/journey/journey_config.ts

```diff
@@ -192,4 +197,12 @@ export class JourneyConfig<CtxExt extends object> {
       new Promise<void>((resolve) => resolve());
     }
   }
+
+  async getAfterStepsFn(ctx: BaseStepCtx & CtxExt) {
+    if (this.#opts.afterSteps) {
+      await this.#opts.afterSteps(ctx);
+    } else {
+      new Promise<void>((resolve) => resolve());
```

Comment:
this branch looks unnecessary - I think behaviour is identical if you remove the else case